herddb-core/src/main/java/herddb/sql/DDLSQLPlanner.java

```diff
@@ -821,6 +832,54 @@ private Statement buildExecuteStatement(String defaultTableSpace, Execute execut
             default:
                 throw new StatementExecutionException("Unsupported command " + execute.getName());
         }
+
+    }
+
+    public Statement QueryConsistencyCheckStatement(String defaultTablespace, String query, List<Object> parameters) {
+        if (query.contains(CalcitePlanner.TABLE_CONSISTENCY_COMMAND) || query.contains(CalcitePlanner.TABLE_CONSISTENCY_COMMAND.toLowerCase())) {
```

Comment:
not "contains"
use only "startsWith" as in other points.
please drop the "toLowercase" stuff. we could improve it in a follow up work